src/checkbox/src/Checkbox.js

```diff
@@ -35,6 +34,42 @@ MinusIcon.propTypes = {
 
 const noop = () => {}
 
+const pseudoSelectors = {
+  _base: '& + div',
+  _disabled: '&[disabled] + div',
+  _hover: '&:not([disabled]):hover + div',
+  _focus: '&:not([disabled]):focus + div',
+  _active: '&:not([disabled]):active + div',
+  _checked: '&:checked + div, &[type=checkbox]:indeterminate + div',
+  _checkedHover:
+    '&:not([disabled]):checked:hover + div, &[type=checkbox]:not([disabled]):indeterminate:hover + div',
+  _checkedActive:
+    '&:not([disabled]):checked:active + div, &[type=checkbox]:not([disabled]):indeterminate:active + div',
+  _checkedDisabled:
+    '&[disabled]:checked + div, &[type=checkbox][disabled]:indeterminate + div'
+}
+
+const internalStyles = {
+  border: '0',
+  clip: 'rect(1px, 1px, 1px, 1px)',
+  height: '1px',
+  overflow: 'hidden',
+  padding: '0',
+  position: 'absolute',
+  whiteSpace: 'nowrap',
+  WebkitFontSmoothing: 'antialiased',
+  textDecoration: 'none',
+  WebkitAppearance: 'none',
+  MozAppearance: 'none',
+  width: '1px',
+  opacity: '0',
+
+  [pseudoSelectors._base]: {
```

Comment:
You don't need this. You can just do `_base`. The pseudoSelectors placeholders aren't replaced until the final step after all the styles (including `internalStyles`) have been merged together.